src/HttpCacheListener.php

```diff
@@ -19,21 +19,21 @@ class HttpCacheListener extends AbstractListenerAggregate
     /**
      * @var array
      */
-    protected $cacheConfig = array();
+    protected $cacheConfig = [];
 
     /**
      * @var array
      */
-    protected $config = array();
+    protected $config = [];
 
     /**
      * @param EventManagerInterface $events
      * @param int                   $priority
      */
-    public function attach(EventManagerInterface $events, $priority = 1)
```

Comment:
Re-instate the `$priority = 1` argument for forwards-compatibility purposes (v3 adds it to the interface).